atlp-rwanda/trojans-ec-fe
# #184760041  Reset Password

### What does this PR do?
Reset Password
### Description of Task to be completed?
In case the users password is forgotten, functionality should exists to reset the password via an associated and email account.
Users should be presented with a reset password link on the login page, there should also be another page for validating token sent via email and new password and confirm password should be inputted and validated.

### How should this be manually tested?
 From the login page you can click on forget password and from there you are enter email to appearing page as you send you check the email to get the message and from the link of email you can reset the password afterAll you return to the login page.
### Any background context you want to provide?
This is a feature
### What are the relevant pivotal tracker stories?
Users should be able to reset the password
### ScreenShot 

<img width="1440" alt="Screenshot 2023-04-25 at 14 52 46" src="https://user-images.githubusercontent.com/117984063/234281813-9f128e9f-5aca-4738-b25b-0ba362f4ea9e.png">

<img width="1440" alt="Screenshot 2023-04-25 at 15 02 45" src="https://user-images.githubusercontent.com/117984063/234284258-a2b6e73d-c950-49e8-95ec-68aa39a61ee0.png">

<img width="1440" alt="Screenshot 2023-04-25 at 15 03 41" src="https://user-images.githubusercontent.com/117984063/234284328-ca22c18b-3193-49f2-9b41-86e22286bf9f.png">


Provide a relevant link to the pivotal tracker.
https://www.pivotaltracker.com/n/projects/2628461
